engines/bastion_katello/app/assets/javascripts/bastion_katello/environments/details/environment.controller.js

```diff
@@ -26,6 +26,7 @@
         $scope.contentTypes = ContentService.contentTypes;
         $scope.errorMessages = [];
         $scope.successMessages = [];
+        $scope.environment = {id: $scope.$stateParams.environmentId};
 
         $scope.environment = Environment.get({id: $scope.$stateParams.environmentId});
```

Comment:
Wouldn't this value immediately get overwritten?   I'm assuming this does solve the issue, its just surprising and if i saw this in the code and was working in this class i would probably delete that line you just added